### PR TITLE
Add CRS:84 as a separate CRS from EPSG:4326

### PR DESCRIPTION
**Pull request by Ardweaden**
`CRS:84` is not identical to `EPSG:4326`, they have switched coordinates (in some cases).

So when requesting FIS with  `EPSG:4326`, but using GeoJSON (which has `CRS:84`), the service switched the coordinates around.

### CODE REVIEW

File: src/crs.ts [hunk 1]
```diff
@@ -35,7 +35,13 @@ export const CRS_EPSG4326: CRS = {
   opengisUrl: 'http://www.opengis.net/def/crs/EPSG/0/4326',
 };
 
-export const CRS_WGS84: CRS = CRS_EPSG4326;
+export const CRS_WGS84: CRS = {
```
Comment (zcernigoj) [inline src/crs.ts:38]: Should we add a comment that this is the same as WGS84 / EPSG:4326, only with flipped coordinates?

**Comment by zcernigoj** [comment]
The change in `src/layer/AbstractSentinelHubV3Layer.ts` should probably be done in `src/layer/AbstractSentinelHubV1OrV2Layer.ts` too.

**Comment by zcernigoj** [comment]
I think there should be a test for the `getStats()` function which would capture the request that the function creates and check the order of the coordinates for different CRSs.

But maybe this can be added later.

**Comment by Ardweaden** [comment]
> The change in `src/layer/AbstractSentinelHubV3Layer.ts` should probably be done in `src/layer/AbstractSentinelHubV1OrV2Layer.ts` too.

@zcernigoj I don't seem to be able to reply directly?

Anyway, thanks, added.

**Comment by Ardweaden** [comment]
> I think there should be a test for the `getStats()` function which would capture the request that the function creates and check the order of the coordinates of the geometry for different CRSs.
> 
> But maybe this can be added later.

@zcernigoj I don't think there's any interesting tests that could be added, except for checking if the specified CRS is in the payload. We don't change the order of coordinates in the FIS request.

Therefore, I instead added 2 stories, one for V3 and one for V12.

File: src/crs.ts (hunk 1 above)
Comment (Ardweaden) [reply src/crs.ts:41]: Sure.

Comment (zcernigoj) [reply src/crs.ts:41]: Thanks. Resolving

**Comment by zcernigoj** [comment]
> > The change in `src/layer/AbstractSentinelHubV3Layer.ts` should probably be done in `src/layer/AbstractSentinelHubV1OrV2Layer.ts` too.
> 
> @zcernigoj I don't seem to be able to reply directly?
> 
> Anyway, thanks, added.

I think Github doesn't that functionality for the comments that are not in the code. 

> > I think there should be a test for the `getStats()` function which would capture the request that the function creates and check the order of the coordinates of the geometry for different CRSs.
> > But maybe this can be added later.
> 
> @zcernigoj I don't think there's any interesting tests that could be added, except for checking if the specified CRS is in the payload. We don't change the order of coordinates in the FIS request.
> 
> Therefore, I instead added 2 stories, one for V3 and one for V12.

:+1:

**Review by zcernigoj — APPROVED** [review]
Can be merged :+1:

**Comment by Ardweaden** [comment]
> I think Github doesn't that functionality for the comments that are not in the code.

@zcernigoj worse UI than EOB search time range.

**Pull request merged**